src/main/resources/org/jpeek/xsd/skeleton.xsd

```diff
@@ -22,6 +22,11 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
+<!--
+@todo #135:30min Skeleton.xsd: add a reference to this schema from the generated
+ skeleton.xml. Schemas from src/resources/org/jpeek/xsd must be referenced in
```

Comment:
@llorllale what `skeleton.xml` has to do with this file? Typo?

Reply:
@yegor256 yes, typo